rackcity/utils/asset_utils.py

```diff
@@ -102,6 +136,9 @@ def copy_asset_to_new_asset_cp(asset_live, change_plan):
             setattr(asset_cp, field.name, getattr(asset_live, field.name))
     asset_cp.save()
 
+    if fields_only:
+        return asset_cp
+
```

Comment:
it seems like the only time you are calling this method with fields_only=True is from another helper method, so it makes it hard for me to determine what a use case in which you would only want the fields but not network connection info? like even if u dont care about network connections, i imagine you would still want mac address info, which lives on networkport